startup/81-dexela.py

```diff
@@ -0,0 +1,69 @@
+import os
+import ophyd
+from hxntools.detectors.dexela import (DexelaDetector, DexelaTiffPlugin)
+from ophyd import (AreaDetector, CamBase, TIFFPlugin, Component as Cpt,
+                    HDF5Plugin, Device, StatsPlugin, ProcessPlugin,
+                    ROIPlugin, EpicsSignal)
+from databroker.assets.handlers import HandlerBase
+from ophyd.areadetector.filestore_mixins import (FileStoreIterativeWrite,
+                                                 FileStoreHDF5IterativeWrite,
+                                                 FileStoreTIFFSquashing,
+                                                 FileStoreTIFF,
+                                                 FileStoreHDF5, new_short_uid,
+                                                 FileStoreBase
+                                                 )
+from ophyd.areadetector import (AreaDetector, PixiradDetectorCam, ImagePlugin,
+                                TIFFPlugin, StatsPlugin, HDF5Plugin,
+                                ProcessPlugin, ROIPlugin, TransformPlugin,
+                                OverlayPlugin)
+from ophyd.areadetector.trigger_mixins import SingleTrigger
+from enum import Enum
+
+
+class XPDDMode(Enum):
+    step = 1
+    fly = 2
+
+
+class XPDDDexelaDetector(SingleTrigger, DexelaDetector):
+    total_points = Cpt(Signal, value=1, doc="The total number of points to be taken")
+    tiff1 = Cpt(DexelaTiffPlugin, 'TIFF1:',
```

Comment:
Why not simply `tiff`? On other beamlines we are not necessarily keeping the same names.

Reply:
sure